packages/calendar/core/src/interfaces/index.ts

```diff
@@ -0,0 +1,8 @@
+import type { Language } from "@prisma/client";
```

Comment:
We can co-locate this to the `calendar-core/src/constants.ts` file so that we we update the constant we have it's interface in the same place.